kde-apps/gwenview/gwenview-23.08.0-r1.ebuild

```diff
@@ -73,10 +75,7 @@ DEPEND="${COMMON_DEPEND}
 	>=dev-qt/qtconcurrent-${QTMIN}:5
 	>=kde-frameworks/kwindowsystem-${KFMIN}:5
 "
-RDEPEND="${COMMON_DEPEND}
-	>=dev-qt/qtimageformats-${QTMIN}:5
-	>=kde-frameworks/kimageformats-${KFMIN}:5
-"
+RDEPEND="${COMMON_DEPEND}"
```

Comment:
Users will simply end up without working image support and left wondering. What's the gain? This is an image viewer after all.

Reply:
plasma-meta would be wrong, since it has nothing to do with gwenview except using similar libraries.

Upstream ships 3 products:
- Plasma
- Gear
- Frameworks

Reply:
Yeah. I wished Portage would properly support optional `RDEPEND`, which means no rebuild on `"RUSE"`, but alas, this is yet to be implemented by someone with more time on their hands in future EAPI.

https://www.gentoo.org/glep/glep-0062.html
https://bugs.gentoo.org/539124

Reply:
But again, the gain from this is questionable. TIFF can be regarded as a standard feature for an image viewer just as JPEG, `qtimageformats` provides very basic image support as well, and `kimageformats` is nicely configurable anyway if you do not wish to have the latest and greatest formats supported.

Reply:
That's true, but there is a build flag to switch and historically this dependency had been lacking behind in porting between major Qt versions with multiple reverse dependencies, and we are approaching such a time once more, so it might become useful yet again.